Subject: [PATCH] Alteração nome do Script
---
Index: Main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Main.py b/Main.py
--- a/Main.py	(revision 53d4d9c667f5705329e999fe35e64697d7fa67e9)
+++ b/Main.py	(revision 28a999b1434bdaf44c06e798029fa9cde562be5e)
@@ -1,5 +1,4 @@
 import subprocess
-
 import re
 
 regular = r'FriendlyName : (\S ?)*[0-9]{14}'
